src/sssom/util.py

```diff
@@ -105,6 +105,59 @@ def with_converter(
             metadata=metadata or get_default_metadata(),
         )
 
+    @classmethod
+    def from_mappings(
+        cls,
+        mappings: List[SSSOM_Mapping],
+        *,
+        converter: HINT = None,
+        metadata: Optional[MetadataType] = None,
+    ) -> "MappingSetDataFrame":
+        """Instantiate from a list of mappings, mapping set metadata, and an optional converter."""
+        metadata = dict(ChainMap(metadata or {}, get_default_metadata()))
```

Comment:
Because this statement is a bit hard to read (I am used to it now), can we factor it into a private `_prepare_metadata()` method?

Reply:
Addressed in 28e67e2